clang/test/CIR/CodeGen/builtins-memory.c

```diff
@@ -145,3 +145,23 @@ void test_memcpy_inline_aligned_buffers(unsigned long long *dst, const unsigned
   // COM: LLVM: call void @llvm.memcpy.inline.p0.p0.i64(ptr align 8 {{%.*}}, ptr align 8 {{%.*}}, i64 4, i1 false)
   __builtin_memcpy_inline(dst, src, 4);
 }
+
+void test_memset_inline(void *dst, int val) {
```

Comment:
Following what I said in the review comment: you could wrap this function with `#ifndef LLVM_SUPPORT` and pass a `-DLLVM_SUPPORT` to the `-emit-llvm` RUN line for now.

Reply:
the https://github.com/llvm/clangir/pull/1135 has been merged ,so we can simply rebase this patch.